save-backend/src/test/kotlin/org/cqfn/save/backend/controller/AgentsControllerTest.kt

```diff
@@ -41,6 +64,46 @@ class AgentsControllerTest {
             .isOk
     }
 
+    @Test
+    @Suppress("TOO_LONG_FUNCTION")
+    fun `check that agent statuses are updated`() {
+        updateAgentStatuses(
+            listOf(
+                AgentStatusDto(LocalDateTime.now(), AgentState.IDLE, "container-2")
```

Comment:
Actually, please use fixed dates here too. Once we get to Jun 2022, this test will start failing :)

Reply:
Fixed